### PR TITLE
Updates to docs from CSM solutions repository

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Data Infrastructure<br>Platform Infrastructure<br>Quality Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Internal Tools<br>Product Partnerships<br>SDKs<br>SMS</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Channels<br>FIX<br>Intelligence<br>Reporting<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Email (Composition and Infrastructure)<br>Ingestion<br>Messaging</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md [hunk 1]
```diff
@@ -666,6 +668,43 @@ This use case captures a list of the categories a user has purchased from, and i
 ```
 {% endraw %}
 
+### Track how many times a custom event occured over the past month {#track}
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md:671]:
```suggestion
### Track how many times a custom event occurred over the past month {#track}
```

File: _docs/_user_guide/data_and_analytics/braze_currents.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ description: "This landing page will tell you more about and guide you to articl
...
+guide_top_text: "Understanding the impact of your engagement strategy is critical in informing your iteration and optimization of your communications with your users. To ensure that this valuable engagement data is tightly integrated with the rest of your operations and help amplify your investment in data science, the Braze platform tracks a wide array of event data from your integration for analysis, retargeting, and other use-cases elsewhere within your own systems. <br> <br>The Currents tool is a real time data stream of your engagement events that is the most robust, yet most granular export out of the Braze platform. It provides you data in an Avro file type to one of our many <a href='https://www.braze.com/docs/user_guide/data_and_analytics/braze_currents/available_partners/'>data partners</a>, empowering you to use the unique and valuable data Braze creates to power your BI and analytics efforts in other best-in-class platforms."
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/braze_currents.md:13]:
```suggestion
guide_top_text: "Understanding the impact of your engagement strategy is critical in informing your iteration and optimization of your communications with your users. To ensure that this valuable engagement data is tightly integrated with the rest of your operations and help amplify your investment in data science, the Braze platform tracks a wide array of event data from your integration for analysis, retargeting, and other use cases elsewhere within your own systems. <br> <br>The Currents tool is a real-time data stream of your engagement events that is the most robust, yet granular export out of the Braze platform. It provides you data in an Avro file type to one of our many <a href='https://www.braze.com/docs/user_guide/data_and_analytics/braze_currents/available_partners/'>data partners</a>, empowering you to use the unique and valuable data Braze creates to power your BI and analytics efforts in other best-in-class platforms."
```

File: _docs/_user_guide/data_and_analytics/custom_data/custom_events.md [hunk 3]
```diff
@@ -11,6 +11,18 @@ description: "This reference article describes custom events and properties, the
 
 Custom events are actions taken by, or updates about, your users. They're best suited for tracking high-value user interactions within your application. Logging a custom event can trigger any number and type of follow-up campaigns, and enables the listed segmentation filters on the recency and frequency of that event.
 
+## Use cases
+
+Some common custom event use cases include:
+- Trigger a campaign or Canvas using [action-based delivery]({{site.baseurl}}/user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/triggered_delivery/) based on a custom event.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_events.md:17]:
```suggestion
- Trigger a campaign or Canvas based on a custom event using [action-based delivery]({{site.baseurl}}/user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/triggered_delivery/).
```

File: _docs/_user_guide/data_and_analytics/custom_data/custom_events.md [hunk 4]
```diff
@@ -11,6 +11,18 @@ description: "This reference article describes custom events and properties, the
...
+- Segment users by how many times they performed a custom event, when the last time the event occurred, etc.
+- Use dashboard [custom event analytics]({{site.baseurl}}/user_guide/data_and_analytics/custom_data/custom_events#custom-event-analytics) to view an aggregate of how often each event occurred
+- Find additional analytics using [funnel]({{site.baseurl}}/user_guide/data_and_analytics/your_reports/funnel_reports/#step-2-select-events-for-funnel-steps) and [retention]({{site.baseurl}}/user_guide/data_and_analytics/your_reports/retention_reports/) reports
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_events.md:20]: Minor callout, if one of your bullets has a period, they should all have a period

```suggestion
- Find additional analytics using [funnel]({{site.baseurl}}/user_guide/data_and_analytics/your_reports/funnel_reports/#step-2-select-events-for-funnel-steps) and [retention]({{site.baseurl}}/user_guide/data_and_analytics/your_reports/retention_reports/) reports.
```

File: _docs/_user_guide/data_and_analytics/custom_data/custom_events.md [hunk 5]
```diff
@@ -11,6 +11,18 @@ description: "This reference article describes custom events and properties, the
...
+- Leverage [persistent entry properties]({{site.baseurl}}/user_guide/engagement_tools/canvas/create_a_canvas/canvas_persistent_entry_properties/) to use metadata from your customer event for personalization in your Canvas steps.
+- Generate more sophisticated analytics with [Currents]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents)
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_events.md:22]:
```suggestion
- Generate more sophisticated analytics with [Currents]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents).
```

File: _docs/_user_guide/data_and_analytics/custom_data/custom_events.md [hunk 6]
```diff
@@ -11,6 +11,18 @@ description: "This reference article describes custom events and properties, the
...
+- Set up Canvas [exception events]({{site.baseurl}}/user_guide/engagement_tools/canvas/create_a_canvas/exception_events#canvas-exception-events) to define when users should not advance to the next step of your Canvas
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_events.md:23]:
```suggestion
- Set up Canvas [exception events]({{site.baseurl}}/user_guide/engagement_tools/canvas/create_a_canvas/exception_events#canvas-exception-events) to define when users should not advance to the next step of your Canvas.
```

File: _docs/_user_guide/data_and_analytics/user_data_collection/best_practices.md [hunk 7]
```diff
@@ -21,22 +21,39 @@ If an unknown user were to view your site and then, at a later date, create an a
...
-- **Client checks internally (recommended)**<br>If you have an external record or database containing the provided user information that exists outside of Braze, reference this at the time of email submission to ensure the email has not already been captured.<br><br>
-- **[/users/export/id]({{site.baseurl}}/api/endpoints/export/user_data/post_users_identifier/) endpoint**<br>Check to see if the returned users array is empty or contains a value. It is not recommended to heavily leverage this endpoint when querying a single user; we apply a rate limit of 2,500 requests per minute to this endpoint. For more information on endpoint rate limits, refer to [Rate limits by request type]({{site.baseurl}}/api/api_limits/#rate-limits-by-request-type).
+
+- **Client checks internally (recommended)**<br>If you have an external record or database containing the provided user information that exists outside of Braze, reference this at the time of email submission or account creation to ensure the information has not already been captured.<br><br>
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/user_data_collection/best_practices.md:28]: "Client checks internally" is this calling the user a client? Or is this referring to client-side data? If it's the former, I'd suggest rewording these two bullets to something along the lines of

- Check internal database
- Use /users/export/id endpoint

If it's the latter, might want to clarify the first bullet point 🤔

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md [hunk 8]
```diff
@@ -717,6 +757,24 @@ This use case will display the current date, month, and year, with the month in
...
+This use case will display an image based on a user's language. Note that this use case has only been tested with images uploaded to Braze.
+
+{% raw %}
+```liquid
+{% if ${language} == 'en' %}
+English image URL (for example, https://cdn-staging.braze.com/appboy/communication/assets/image_assets/images/60aecba96a93150c749b4d57/original.png?1622068137)
+{% elsif ${language} == 'ru' %}
+Russian iamge URL
+{% elsif ${language} == 'es' %}
+Spanich image URL
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md:771]:
```suggestion
Spanish image URL
```

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md [hunk 9]
```diff
@@ -717,6 +757,24 @@ This use case will display the current date, month, and year, with the month in
...
+### Display an image based on a user's language {#language-image-display}
+
+This use case will display an image based on a user's language. Note that this use case has only been tested with images uploaded to Braze.
+
+{% raw %}
+```liquid
+{% if ${language} == 'en' %}
+English image URL (for example, https://cdn-staging.braze.com/appboy/communication/assets/image_assets/images/60aecba96a93150c749b4d57/original.png?1622068137)
+{% elsif ${language} == 'ru' %}
+Russian iamge URL
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md:769]:
```suggestion
Russian image URL
```

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md [hunk 10]
```diff
@@ -717,6 +757,24 @@ This use case will display the current date, month, and year, with the month in
 ```
 {% endraw %}
 
+### Display an image based on a user's language {#language-image-display}
+
+This use case will display an image based on a user's language. Note that this use case has only been tested with images uploaded to Braze.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/liquid_use_cases.md:762]: Might be good to add more detail here, but feel free to discard 🤔 
```suggestion
This use case will display an image based on a user's language. Note that this use case has only been tested with images uploaded to the Braze Media Library.
```

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Looks really good! Left a few minor comments for some grammar fixes and clarifications.

File: _docs/_user_guide/data_and_analytics/braze_currents.md [hunk 11]
```diff
@@ -52,6 +52,6 @@ There’s so much more you can do with event data, accessed by Currents. Trust u
 
 ## Access Currents
 
-A Currents connector is already included in many of our pro- and enterprise-level packages. If you’re interested using Currents, reach out to your account manager. Your account manager and our data specialists can assist in your Currents [setup and integration]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/setting_up_currents/).
+A Currents connector is already included in many of our pro and enterprise-level packages. If you’re interested using Currents, reach out to your account manager. Your account manager and our data specialists can assist in your Currents [setup and integration]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/setting_up_currents/).
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/braze_currents.md:55]:
```suggestion
A Currents connector is already included in many of our pro and enterprise-level packages. If you’re interested using Currents, reach out to your account manager. Your account manager and our data specialists can assist in your [Currents setup and integration]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/setting_up_currents/).
```

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md [hunk 12]
```diff
@@ -17,6 +17,8 @@ When stored in Braze, these characteristics can be used to build out audience se
 
 To create and manage custom attributes in the dashboard, go to **Manage Settings** > **Custom Attributes**. From this page, you can view, manage, or blocklist existing custom attributes, or create a new one. If you block a custom attribute, no data will be collected regarding that attribute, existing data will be unavailable unless reactivated, and blocklisted attributes will not show up in filters.
 
+If you would like to remove custom attributes from user profiles, set the value to "null" in your API request to the [/users/track]({{site.baseurl}}/api/endpoints/user_data/post_user_track#user-track) endpoint.
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md:20]:
```suggestion
If you would like to remove custom attributes from user profiles, set the value to "null" in your API request to the [`/users/track`]({{site.baseurl}}/api/endpoints/user_data/post_user_track#user-track) endpoint.
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
looks great! left two small suggestions

File: _docs/_user_guide/data_and_analytics/user_data_collection/best_practices.md (hunk 7 above)
Comment (KellieHawks) [inline _docs/_user_guide/data_and_analytics/user_data_collection/best_practices.md:28]:
```suggestion
- **Checks internal database (recommended)**<br>If you have an external record or database containing the provided user information that exists outside of Braze, reference this at the time of email submission or account creation to ensure the information has not already been captured.<br><br>
```

**Review by lydia-xie — APPROVED** [review]

**Review by bre-fitzgerald — APPROVED** [review]
